strict/src/Data/Strict/Either.hs

```diff
@@ -6,6 +6,18 @@
 {-# LANGUAGE DeriveGeneric      #-}
 #endif
 
+#if MIN_VERSION_base(4,9,0)
+#define LIFTED_FUNCTOR_CLASSES 1
+#else
+#if MIN_VERSION_transformers(0,5,0)
+#define LIFTED_FUNCTOR_CLASSES 1
+#else
```

Comment:
You can shorten `#else #if` to `#elif` then we can have fewer `#endif` at the end.

(Same for the other files.)

Reply:
We rather not. See https://github.com/isomorphism/these/pull/146

Reply:
This sucks, but OK. :(

Reply:
I had the very same feeling when looking at the patch and reason with `these`.

CentOS-7:
- Full updates: Q4 2020
- Maintenance Update: June 30th, 2024

Just four more years!